js/app.js

```diff
@@ -0,0 +1,271 @@
+// 1. Easy Going
+
+// for (let i = 1; i <= 20; i++) {
+// 	console.log(i);
+// }
+
+// 2. Get Even
+
+// for (let a = 0; a <= 200; a +=2) {
+// 	console.log(a);
+// }
+
+// 3. Excited Kitten
+
+// let cat1 = "...human...why you taking pictures of me?..."
+// let cat2 = "...the catnip made me do it..."
+// let cat3 = "...why does the red dot always get away..."
+
+// for (let b = 0; b <= 20; b++) {
+// 	if ((b % 2)== 0) {
+// 		let random1 = (Math.floor(Math.random() * 10 + 1))
+// 		if (random1 < 3) {
+// 			console.log(cat1)
+// 		}
+// 		else if (random1 >= 3 && <= 6){
+// 			console.log(cat2)
+// 		}
+// 		else if (random1 >6){
+// 			console.log(cat3)
+// 		} 
+// 	}
+// 	else {
+// 		console.log("Love me, pet me! HSSSSSS!")
+// 	}
+// }
+
+// 4. Fizz Buzz
+
+// for (let c = 1; c <= 100; c++){
+// 	if ((c % 3) == 0){
+// 		console.log("Fizz")
+// 	}
+// 	else if((c % 5) == 0){
+// 		console.log("Buzz")
+// 	}
+// 	else if(((c % 5) == 0) && ((c % 3) == 0)) {
+// 		console.log("FizzBuzz")
+// 	}
```

Comment:
i%3 === 0 && i % 5 === 0 condition needs to go first else it is never executed when numbers are divisible by both.